allennlp/tests/commands/train_test.py

```diff
@@ -318,3 +318,143 @@ def test_vocab_extended_model_with_transferred_embedder_is_loadable(self):
         # Check that such a dumped model is loadable
         # self.serialization_dir = self.TEST_DIR / 'fine_tune'
         load_archive(str(self.TEST_DIR / "train" / "model.tar.gz"))
+
+
+class TestFineTune(AllenNlpTestCase):
```

Comment:
Why is this a separate class?  Just have these be additional tests on the main class, unless this is because you really want different setup for these tests.  If that's true, then add a comment saying so.